CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 
 ### Added
 
+### Changed
+
+### Fixed
+
+
+## [3.6.3] - 2024-01-16
+
+### Added
```

Comment:
I made one minor change to this file, just to move the "unreleased" stuff that was sitting in main to the "3.6.3" section, since we're going to do a release